app/components/Analyst/Project/ProjectInformation/widgets/ExcelImportFileWidget.tsx

```diff
@@ -187,7 +187,10 @@ const SowImportFileWidget: React.FC<SowImportFileWidgetProps> = ({
   const [createAttachment, isCreatingAttachment] = useCreateAttachment();
   const [deleteAttachment, isDeletingAttachment] = useDeleteAttachment();
   const [isImporting, setIsImporting] = useState(false);
-  const [isValidSow, setIsValidSow] = useState(false);
+  const [isValidExcel, setIsValidExcel] = useState(false);
+  const excelImportOptions = options?.excelImport;
+  const successHeading = excelImportOptions['successHeading']?.toString();
```

Comment:
nit: I think the `toString()` here can cause our code to be brittle. If at one point the value is a number which would pass the `?` check we'll get an error. I'm cool with leaving it in since I doubt we'll ever have the success heading be a number but can cause an error down the line.

I think it might also produce weird behaviour if we provide it with JSX, probably get a `[object Object]` as the heading.

Could throw some checks in the `renderStatusLabel` instead?